File: converti.py
import pandas as pd

df = pd.read_csv("svincolati.csv", encoding="utf-8")
df.to_parquet("svincolati.parquet", index=False, compression="snappy")

print(f"Righe: {len(df)}")
print(f"Colonne: {list(df.columns)}")
print("Fatto!")

File: converti.js
const fs = require("fs");

// Leggi il CSV
const csv = fs.readFileSync("svincolati.csv", "utf-8");
const lines = csv.split("\n");
const headers = lines[0].split(",");

console.log(`Righe: ${lines.length - 1}`);
console.log(`Colonne: ${headers}`);